Source/WebKit/GPUProcess/graphics/WebGPU/RemoteBuffer.cpp

```diff
@@ -76,8 +76,8 @@ void RemoteBuffer::mapAsync(WebCore::WebGPU::MapModeFlags mapModeFlags, WebCore:
 
 void RemoteBuffer::getMappedRange(WebCore::WebGPU::Size64 offset, std::optional<WebCore::WebGPU::Size64> size, CompletionHandler<void(std::optional<Vector<uint8_t>>&&)>&& callback)
 {
-    protectedBacking()->getMappedRange(offset, size, [&] (auto mappedRange) {
-        m_isMapped = true;
+    protectedBacking()->getMappedRange(offset, size, [protectedThis = Ref<RemoteBuffer>(*this), &callback] (auto mappedRange) {
```

Comment:
I think:
```
Ref<RemoteBuffer>(*this)
```
can simply be:
```
Ref { *this }
```

Reply:
You are right, fixed. Maybe this should also be a style checker thing since we are inconsistent in this area.